plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanelTest.java

```diff
@@ -163,28 +169,35 @@ public void testUncheckStopPreviousVersionButtonWhenDisabled() {
   }
 
   @Test
-  public void testProjectSavedInPreferencesSelected() throws ProjectRepositoryException {
+  public void testProjectSavedInPreferencesSelected()
+      throws ProjectRepositoryException, InterruptedException, BackingStoreException {
     IEclipsePreferences node =
         new ProjectScope(project).getNode(StandardDeployPreferences.PREFERENCE_STORE_QUALIFIER);
     node.put("project.id", "projectId1");
     node.put("account.email", EMAIL_1);
     initializeProjectRepository();
     when(loginService.getAccounts()).thenReturn(new HashSet<>(Arrays.asList(account1, account2)));
     deployPanel = createPanel(true /* requireValues */);
+    deployPanel.latestGcpProjectQueryJob.join();
+    node.clear();
+
+    ProjectSelector projectSelector = getProjectSelector();
```

Comment:
With https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1653#discussion_r107927606, it will never return `null`, so we are covered.